### ISSUE TITLE
New changelogs should mention which version of core's changelog it tracks

### ISSUE THREAD

**Issue by OnGle** [turnkeylinux/tracker#1554]
As versions are now somewhat disjointed it is no longer always clear which "core changelog" an appliance is refering too in the changelogs. As for example a v16.2 *could* be built off a v16.0 core. As such the following changelog snippet should be used

```
  * Note: Please refer to turnkey-core's $VERSION changelog for changes common to all
    appliances. Here we only describe changes specific to this appliance.
```

where `$VERSION` is `$MAJOR.$MINOR` for example:
- 16.0
- 16.1
- 17.1
etc.

-----
(@JedMeister)

**Comment by JedMeister** [comment]
Attention @qq7 - Just wanting to draw your attention to this for v16.1 updates. Please treat the suggestion posted by @OnGle as the canonical text for providing as the last point within all v16.1 appliances' changelogs. i.e. copy/paste this as the last part of an applaince changelog.

**Comment by OnGle** [comment]
@JedMeister not sure what to do with this. Presumably this applies to all subsequent releases, and won't ever be *finished*, thoughts?

**Comment by JedMeister** [comment]
@OnGle 

> Presumably this applies to all subsequent releases, and won't ever be _finished_

Yes, good point! Perhaps what is need to close this issue is add it into the TKLDev docs? IIRC there is a doc that covers writing changelogs and changelog entries.

Also, perhaps we need to have a policy doc somewhere that covers this too?

Thoughts?

**Comment by JedMeister** [comment]
IMO this is a "nice to have" ATM - especially when we have such a large backlog of work. So I'm going to re-target it to v20.0. If we get time to address it sooner all well and good...

I think the way to resolve this is to publish the HEAD commit ID of common when each appliance is built. Including it in the relevant appliance metadata dir on the mirror would probably make the most sense, e.g. http://mirror.turnkeylinux.org/turnkeylinux/metadata/turnkey-openvpn/18.1-bookworm-amd64/
